text/0000-trits-t5b1/0000-trits-t5b1.md

```diff
@@ -0,0 +1,119 @@
+    [ 1, 1, -1, -1, -1 ],  [ -1, -1, 0, -1, -1 ], [ 0, -1, 0, -1, -1 ],   [ 1, -1, 0, -1, -1 ],  [ -1, 0, 0, -1, -1 ],
+    [ 0, 0, 0, -1, -1 ],   [ 1, 0, 0, -1, -1 ],   [ -1, 1, 0, -1, -1 ],   [ 0, 1, 0, -1, -1 ],   [ 1, 1, 0, -1, -1 ],
+    [ -1, -1, 1, -1, -1 ], [ 0, -1, 1, -1, -1 ],  [ 1, -1, 1, -1, -1 ],   [ -1, 0, 1, -1, -1 ],  [ 0, 0, 1, -1, -1 ],
+    [ 1, 0, 1, -1, -1 ],   [ -1, 1, 1, -1, -1 ],  [ 0, 1, 1, -1, -1 ],    [ 1, 1, 1, -1, -1 ],   [ -1, -1, -1, 0, -1 ],
+    [ 0, -1, -1, 0, -1 ],  [ 1, -1, -1, 0, -1 ],  [ -1, 0, -1, 0, -1 ],   [ 0, 0, -1, 0, -1 ],   [ 1, 0, -1, 0, -1 ],
+    [ -1, 1, -1, 0, -1 ],  [ 0, 1, -1, 0, -1 ],   [ 1, 1, -1, 0, -1 ],    [ -1, -1, 0, 0, -1 ],  [ 0, -1, 0, 0, -1 ],
+    [ 1, -1, 0, 0, -1 ],   [ -1, 0, 0, 0, -1 ],   [ 0, 0, 0, 0, -1 ],     [ 1, 0, 0, 0, -1 ],    [ -1, 1, 0, 0, -1 ],
+    [ 0, 1, 0, 0, -1 ],    [ 1, 1, 0, 0, -1 ],    [ -1, -1, 1, 0, -1 ],   [ 0, -1, 1, 0, -1 ],   [ 1, -1, 1, 0, -1 ],
+    [ -1, 0, 1, 0, -1 ],   [ 0, 0, 1, 0, -1 ],    [ 1, 0, 1, 0, -1 ],     [ -1, 1, 1, 0, -1 ],   [ 0, 1, 1, 0, -1 ],
+    [ 1, 1, 1, 0, -1 ],    [ -1, -1, -1, 1, -1 ], [ 0, -1, -1, 1, -1 ],   [ 1, -1, -1, 1, -1 ],  [ -1, 0, -1, 1, -1 ],
+    [ 0, 0, -1, 1, -1 ],   [ 1, 0, -1, 1, -1 ],   [ -1, 1, -1, 1, -1 ],   [ 0, 1, -1, 1, -1 ],   [ 1, 1, -1, 1, -1 ],
+    [ -1, -1, 0, 1, -1 ],  [ 0, -1, 0, 1, -1 ],   [ 1, -1, 0, 1, -1 ],    [ -1, 0, 0, 1, -1 ],   [ 0, 0, 0, 1, -1 ],
+    [ 1, 0, 0, 1, -1 ],    [ -1, 1, 0, 1, -1 ],   [ 0, 1, 0, 1, -1 ],     [ 1, 1, 0, 1, -1 ],    [ -1, -1, 1, 1, -1 ],
+    [ 0, -1, 1, 1, -1 ],   [ 1, -1, 1, 1, -1 ],   [ -1, 0, 1, 1, -1 ],    [ 0, 0, 1, 1, -1 ],    [ 1, 0, 1, 1, -1 ],
+    [ -1, 1, 1, 1, -1 ],   [ 0, 1, 1, 1, -1 ],    [ 1, 1, 1, 1, -1 ],     [ -1, -1, -1, -1, 0 ], [ 0, -1, -1, -1, 0 ],
+    [ 1, -1, -1, -1, 0 ],  [ -1, 0, -1, -1, 0 ],  [ 0, 0, -1, -1, 0 ],    [ 1, 0, -1, -1, 0 ],   [ -1, 1, -1, -1, 0 ],
+    [ 0, 1, -1, -1, 0 ],   [ 1, 1, -1, -1, 0 ],   [ -1, -1, 0, -1, 0 ],   [ 0, -1, 0, -1, 0 ],   [ 1, -1, 0, -1, 0 ],
+    [ -1, 0, 0, -1, 0 ],   [ 0, 0, 0, -1, 0 ],    [ 1, 0, 0, -1, 0 ],     [ -1, 1, 0, -1, 0 ],   [ 0, 1, 0, -1, 0 ],
+    [ 1, 1, 0, -1, 0 ],    [ -1, -1, 1, -1, 0 ],  [ 0, -1, 1, -1, 0 ],    [ 1, -1, 1, -1, 0 ],   [ -1, 0, 1, -1, 0 ],
+    [ 0, 0, 1, -1, 0 ],    [ 1, 0, 1, -1, 0 ],    [ -1, 1, 1, -1, 0 ],    [ 0, 1, 1, -1, 0 ],    [ 1, 1, 1, -1, 0 ],
+    [ -1, -1, -1, 0, 0 ],  [ 0, -1, -1, 0, 0 ],   [ 1, -1, -1, 0, 0 ],    [ -1, 0, -1, 0, 0 ],   [ 0, 0, -1, 0, 0 ],
+    [ 1, 0, -1, 0, 0 ],    [ -1, 1, -1, 0, 0 ],   [ 0, 1, -1, 0, 0 ],     [ 1, 1, -1, 0, 0 ],    [ -1, -1, 0, 0, 0 ],
+    [ 0, -1, 0, 0, 0 ],    [ 1, -1, 0, 0, 0 ],    [ -1, 0, 0, 0, 0 ]
+]
+```
+
+Because the memory footprint (`1.2kb`) of this `LUT` can be too high for some very limited systems, we also present algorithms that do not use it.
+
+## Encoding
+
+<!-- TODO -->
+
+## Decoding
+
+### With LUT
+
+Decoding a `byte` into `5` `trits` with the help of the `LUT` is fairly simple:
```

Comment:
How should decoding behave when we need to decode a `byte` into less than `5` trits (for example, for 27 encoded trits there are only 2 trits encoded in the tail byte)? Just decode `byte` into `5` trits and drop unneeded trits?

Reply:
You can encode just 2 trits to a byte,  just consider the 3 other trits as 0